fr/integrations/plugin-packs/procedures/network-switchs-symbol-wing-snmp.md

```diff
@@ -0,0 +1,178 @@
+---
+id: network-switchs-symbol-wing-snmp
+title: Symbol WiNG switch SNMP
+---
+
+## Contenu du Pack
+
+### Modèles
+
+Le Plugin Pack Centreon Symbol WiNG switch SNMP apporte 1 modèle d'hôte:
+* Net-Symbol-Wing-SNMP
+
+Il apporte les Modèles de Services suivants :
+
+| Service Alias        | Service Template                          | Default | Discovery |
+|:---------------------|:------------------------------------------|:--------|:----------|
+| Systems              | Net-Symbol-Wing-Systems-SNMP              | X       |           |
+| Traffic-Generic-Id   | Net-Symbol-Wing-Traffic-Generic-Id-SNMP   |         |           |
+| Traffic-Generic-Name | Net-Symbol-Wing-Traffic-Generic-Name-SNMP |         | X         |
+| Traffic-Global       | Net-Symbol-Wing-Traffic-Global-SNMP       |         |           |
+| Packet-Errors        | Net-Symbol-wing-Packet-Errors-Global-SNMP |         | X         |
+
+### Règles de découverte
+
+| Rule name                               | Description |
+|:----------------------------------------|:------------|
+| Net-Symbol-Wing-SNMP-Traffic-Name       |             |
+| Net-Symbol-Wing-SNMP-Packet-Errors-Name |             |
+
+### Métriques & statuts collectés
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Systems-->
+
+* Global
+
+| Metric name         | Description       | Unit  |
+|:--------------------|:------------------|:------|
+| devices.total.count | total devices: %s | count |
+
+* Par *cpu*
+
+| Metric name                    | Description   | Unit |
+|:-------------------------------|:--------------|:-----|
+| cpu.utilization.1m.percentage  | %.2f %% (1m)  | %    |
+| cpu.utilization.5m.percentage  | %.2f %% (5m)  | %    |
+| cpu.utilization.15m.percentage | %.2f %% (15m) | %    |
+
+* Par *memory*
+
+| Metric name                    | Description | Unit |
+|:-------------------------------|:------------|:-----|
+| device.memory.usage.bytes      |             | B    |
+| device.memory.free.bytes       |             | B    |
+| device.memory.usage.percentage |             | %    |
```

Comment:
Same comment for the metrics description in the english files

Reply:
done